src/electron/main.tsx

```diff
@@ -110,7 +110,7 @@ if (!handleSquirrelEvent()) {
   // if any of these checks return false, don't do any other initialization since the app is quitting
   if (ensureSingleInstance() && ensureCorrectEnvironment()) {
     // this needs to happen early in startup so all processes share the same global config
-    chiaConfig.loadConfig(chiaEnvironment.getChiaVersion());
+    chiaConfig.loadConfig('mainnet');
```

Comment:
how it will work with testnet?

Reply:
CHIA_ROOT will have to be set, which is the case for this bit of code already.